content/intro-to-storybook/ember/en/data.md

```diff
@@ -147,9 +139,8 @@ export default class TaskController extends Controller {
 }
 ```
 
-And we use our earlier created presentational `<TaskList>` component, mount it
-at the route template and connect it with the actions in the controller and the
-data from the route.
+And finally we create a new file called `template.hbs` inside the `tasks`
+folder, in which we'll add the presentational `<TaskList>` component we've created in the [previous chapter](./composite-component):
```

Comment:
@gossi if you don't mind update the link from (./composite-component) to (/ember/en/composite-component/), to avoid issues with the routing.

Reply:
Ah thanks, that puzzled me how to do the link (I didn't looked into the other files tbh 🙈).